orttraining/orttraining/python/training/_ortmodule_utils.py

```diff
@@ -5,58 +5,85 @@
 
 from . import _utils
 
-import onnxruntime
 from onnxruntime.capi.onnxruntime_inference_collection import OrtValue
 from onnxruntime.capi import _pybind_state as C
 
 import torch
 from torch.utils.dlpack import from_dlpack, to_dlpack
 from torch.utils.cpp_extension import load_inline
 
+from enum import IntEnum
```

Comment:
I think this should be a part of ```_ortmodule_logger.py```. Maybe the merge conflict resolution did not apply well here.